mantidimaging/core/io/loader/test/loader_test.py

```diff
@@ -107,44 +66,44 @@ def test_create_loading_parameters_for_file_path(self, mock_read_in_file_informa
         sample = lp.sample
         self.assertEqual(image_format, sample.format)
         self.assertEqual(None, sample.indices)
-        self.assertEqual(tomo_dir, sample.input_path)
-        self.assertEqual(mock_find_and_verify_sample_log.return_value, sample.log_file)
-        self.assertEqual(tomo_prefix, sample.prefix)
+        self._files_equal("/b/Tomo", sample.input_path)
+        self._files_equal("/b/Tomo_log.txt", sample.log_file)
+        self._files_equal("/b/Tomo/Tomo", sample.prefix)
```

Comment:
Minor: Could we use parameterized to shorten this tests length?

Reply:
Possibly, but it should get nicer once create_loading_parameters_for_file_path is cleaned up.

Reply:
Happy to hold off to see if it gets nicer as you say once `create_loading_parameters_for_file_path` has been cleaned up